cassandra/1.cql

```diff
@@ -12,7 +12,8 @@ CREATE TABLE devgym.task (
   year TIMESTAMP,
   type TEXT,
   timeuuid TIMEUUID,
-  task_description TEXT,
+  name TEXT,
```

Comment:
I mean to have a Name is very desirable, but it CAN be optional as we just can use description (alright part of it) as name if user do not want to provide one